src/main/webapp/WEB-INF/views/admin/oss/edit-js.jsp

```diff
@@ -506,7 +506,9 @@
 						contentType : 'application/json',
 						success : function(data){
 							var length = data.ossList.length;
-							if (length > 1) {
+							if (length == 1) {
+								alertify.alert('At least two OSS versions are required.', function(){})
```

Comment:
; is missing

Reply:
Thank you for leaving comments 
I modified it 😄